<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Jadwal - Tambah</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="jamMasukLabel">
      <property name="text">
       <string>Jam Masuk</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTimeEdit" name="jamMasukTimeEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QTimeEdit" name="jamPulangTimeEdit"/>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="simpanButton">
        <property name="text">
         <string>Simpan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="batalButton">
        <property name="text">
         <string>Batal</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="jamPulangLabel">
      <property name="text">
       <string>Jam Pulang</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
